autils/network/ports.py

```
@@ -0,0 +1,221 @@
+def is_port_free(port, address):
+    """This method is deprecated. Please use is_port_available().
+
+    :param port: Port value to check.
+    :type port: int
+    :param address: Address to use this port.
+    :type address: str
+    :return: True if the port is available, False otherwise.
+    :rtype: bool
+    """
+
+    warnings.warn("deprecated, use is_port_available() instead.", DeprecationWarning)
```

Ruff (0.12.2)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)